Jamfile

```diff
@@ -247,10 +247,6 @@ rule warnings ( properties * )
 		result += <cflags>-Wno-c++11-long-long ;
 		result += <cflags>-Wno-variadic-macros ;
 
-# in C++98 mode there's no way to silence this warning
-# in the code (without final)
-		result += <cflags>-Wno-non-virtual-dtor ;
```

Comment:
I would expect any place where this warning would be triggered can be marked as final now, since we have C++11 in master. so, this seems good